packages/tupaia-web/src/features/Dashboard/ExportDashboard.tsx

```diff
@@ -50,14 +50,18 @@ const ButtonGroup = styled.div`
 interface ExportDashboardProps {
   isOpen: boolean;
   onClose: () => void;
-  dashboardItems?: DashboardItem[];
+  dashboardItems: DashboardItem[];
```

Comment:
My only comment here would be when the dashboard items are loading they may be undefined

Reply:
I added a default `dashboardItems = []` in the component args, so it should always have a value now. Seems like we were doing something similar before with defaulting to an empty list if the `dashboardItems` were `undefined` and I felt this was a little cleaner 👍